docs/src/dev/runtime_config.md

```diff
@@ -78,6 +56,7 @@ then it is implicitly interpreting.
 This variable is only available when building `ykrt` with the
 `yk_jitstate_debug` Cargo feature enabled.
 
+
```

Comment:
These should have been a separate commit, but I'll let it slide ;)

Reply:
I plead ignorance.